tchannel/tornado/message_factory.py

```diff
@@ -97,7 +97,7 @@ def build_raw_request_message(self, request, args, is_completed=False):
         if request.state == StreamState.init:
             message = CallRequestMessage(
                 flags=request.flags,
-                ttl=request.ttl * 1000,
+                ttl=request.ttl,
```

Comment:
No, you misunderstand me. The point is to *always* use unit names in time variables regardless of how many you're mixing together. That way it's clear what kind of value that variable contains/expects. `ttl_ms` and `ttl_secs` were examples. In this case you'd probably have `ttl_ms` everywhere.